guides/saas/getting-started.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Getting started with Coder's hosted beta
+description: Get started with a hosted Coder deployment.
+---
+
+To help you get started, we've put together this guide to help you understand
+how the Coder hosted beta works and how to access your deployment.
+
+## What is Coder hosted beta?
+
+The Coder hosted beta is a hybrid cloud offering of Coder's self-hosted
+platform. With this model, Coder hosts the control plane, and you host your
+compute and development workspaces. The benefits include:
+
+- Automatic upgrades
+- No installation or DNS configuration
```

Comment:
```suggestion
- DNS or TLS configuration
```

They still have to `install` a provider so I'm not sure "no installatino" is valid.